eth-bridge/contracts/src/Bridge.sol

```diff
@@ -59,37 +36,6 @@ error AlreadyVoted();
 /// Transferred amount is less than configured minimum
 error TooSmallAmount();
 
-/// @title Interface with events emitted by the bridge
```

Comment:
Extracted to `BridgeEvents.sol`, as new solhint version doesn't like multiple contracts in one file.